controller/external.go

```diff
@@ -17,11 +25,189 @@ func StartAWSSessions(secretsCache wranglerv1.SecretCache, spec eksv1.EKSCluster
 	}
 	return sess, eks.New(sess), nil
 }
+*/
 
 // NodeGroupIssueIsUpdatable checks to see the node group can be updated with the given issue code.
 func NodeGroupIssueIsUpdatable(code string) bool {
-	return code == eks.NodegroupIssueCodeAsgInstanceLaunchFailures ||
-		code == eks.NodegroupIssueCodeInstanceLimitExceeded ||
-		code == eks.NodegroupIssueCodeInsufficientFreeAddresses ||
-		code == eks.NodegroupIssueCodeClusterUnreachable
+	return code == string(ekstypes.NodegroupIssueCodeAsgInstanceLaunchFailures) ||
+		code == string(ekstypes.NodegroupIssueCodeInstanceLimitExceeded) ||
+		code == string(ekstypes.NodegroupIssueCodeInsufficientFreeAddresses) ||
+		code == string(ekstypes.NodegroupIssueCodeClusterUnreachable)
+}
+
+// buildUpstreamClusterState
```

Comment:
good to have a proper godoc if we have func name already here

Reply:
good idea, fixed